src/django/pfb_analysis/management/commands/update_status.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management.base import BaseCommand
+
+from pfb_analysis.models import AnalysisJobStatusUpdate, AnalysisJob
+
+
+class Command(BaseCommand):
+    help = "Update status during an analysis job"
+
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument('job_id')
+        parser.add_argument('status')
+        parser.add_argument('step', nargs='?')
```

Comment:
Not sure `nargs='?'` makes sense here if a default is not provided. Making all these arguments required seems sensible.